packages/front-end/components/Ideas/ImpactModal.tsx

```diff
@@ -23,12 +23,9 @@ const ImpactModal: FC<{
     defaultValues: {
       metric: estimate?.metric || metrics[0]?.id || "",
       segment: estimate?.segment || "",
-      // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-      userAdjustment: idea.estimateParams?.userAdjustment || 100,
-      // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-      numVariations: idea.estimateParams?.numVariations || 2,
-      // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-      improvement: idea.estimateParams?.improvement || 10,
+      userAdjustment: idea?.estimateParams?.userAdjustment || 100,
```

Comment:
I think instead we can just change the `idea` prop for this component to not be optional.  This component is only used 1 place and `idea` is non-null there.